fdbserver/WorkerInterface.actor.h

```diff
@@ -149,6 +149,7 @@ struct ClusterControllerFullInterface {
 	RequestStream<struct RegisterWorkerRequest> registerWorker;
 	RequestStream<struct GetWorkersRequest> getWorkers;
 	RequestStream<struct RegisterMasterRequest> registerMaster;
+	RequestStream<struct UpdateWorkerHealthRequest> updateWorkerHealth;
```

Comment:
Careful about adding new endpoint to release branch. 
It has to be appended to the request steam list.